src/event.bif

```diff
@@ -906,3 +906,21 @@ event unknown_protocol%(analyzer_name: string, protocol: count, first_bytes: str
 ##
 ## pkt: Data for the unprocessed packet
 event packet_not_processed%(pkt: pcap_packet%);
+
+## Generated for each new interface in a pcapng file.
+##
+## intf: A record containing data about the interface.
+event pcapng_new_interface%(intf: Pcapng::Interface%);
+
+## Generated when a section header block is found in a pcapng file. This contains various
+## fields about the file.
+##
+## info: A record containing the fields from the section header block.
+event pcapng_file_info%(info: Pcapng::FileInfo%);
```

Comment:
With Spicy parsers we started moving to a model where events are also module scoped. E.g. `Pcapng::new_interface()`. But I like the proposed `pcapng_<event>` more here. Mostly wanted to mention it if we ever should settle on a "proper" way for the future.